### PR TITLE
Fix installing packages on MODX 3.x

### PR DESCRIPTION
**Pull request by muzzwood**
Need to test this on different PHP versions and make sure it still works in MODX 2.x

### CODE REVIEW

File: src/Command/InstallPackageCommand.php [hunk 1]
```diff
@@ -286,9 +289,16 @@ private function download($packageName, $provider, $options = []) {
...
+        if ($this->isMODX3) {
+            $error = $response->getStatusCode() !== 200;
+        }
+        else {
+            $error = $response->isError();
+        }
 
-            $foundPkg = simplexml_load_string($response->response);
+        if (!$error) {
+            $responseBody = $response instanceof Response ? $response->getBody() : $response->response;
```
Comment (Mark-H) [inline src/Command/InstallPackageCommand.php:300]: Shouldn't this be `$response->getBody()->getContents()`? 

(Maybe it is automatically getting cast to a string, but getBody() returns a stream object IIRC)

File: src/BaseCommand.php [hunk 2]
```diff
@@ -50,6 +59,29 @@ public function initialize(InputInterface $input, OutputInterface $output)
...
+
+            $modxVersion = $this->modx->getVersionData();
+            if (version_compare($modxVersion['full_version'], '3.0.0-dev', '>=')) {
+                $this->isMODX3 = true;
+            }
+
+            // If we're on MODX 3, set up some class aliases.
+            if ($this->isMODX3) {
+                class_alias(modTransportProvider::class, 'modTransportProvider');
+                class_alias(modTransportProvider::class, 'transport.modTransportProvider');
```
Comment (Mark-H) [inline src/BaseCommand.php:71]: Hmm, I don't think PHP classes can use the MODX syntax like that with `transport.`. Wouldn't it be easier to adjust the calls/type checks? I assumed the problem was just in the modTransportProvider type checks?

File: src/Command/InstallPackageCommand.php (hunk 1 above)
Comment (muzzwood) [reply src/Command/InstallPackageCommand.php:300]: Yeah that must've been cast as a string. Thanks, updated to use getBody() 👍🏻

File: src/BaseCommand.php (hunk 2 above)
Comment (muzzwood) [reply src/BaseCommand.php:71]: You're right, and they were unnecessary anyway. Aliasing just the base class name (e.g. `modTransportProvider`) was enough.

**Pull request merged**